.github/workflows/format.yml

```diff
@@ -19,4 +19,22 @@ jobs:
           LEGO1/*.cpp LEGO1/*.h \
           LEGO1/realtime/*.cpp LEGO1/realtime/*.h \
           LEGO1/tgl/*.h \
-          LEGO1/viewmanager/*.cpp LEGO1/viewmanager/*.h
\ No newline at end of file
+          LEGO1/viewmanager/*.cpp LEGO1/viewmanager/*.h
+
+  python-format:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Build isledecomp library
```

Comment:
This step name is incorrect.

Reply:
Also, there are 2 `pip install` invocations: here, and at the start of the next step.
Can/should we combine them?

Reply:
Should be all fixed up now, tidied it up across all the .yml files